src/app/GitUI/Editor/Diff/DiffHighlightService.cs

```diff
@@ -13,6 +13,9 @@ namespace GitUI.Editor.Diff;
 /// </summary>
 public abstract class DiffHighlightService : TextHighlightService
 {
+    private static readonly Color _addedBackColor = AppColor.AnsiTerminalGreenBackNormal.GetThemeColor();
```

Comment:
The theme color is basically a dictionary lookup if I remember correctly. Is this needed?

Reply:
These fields are used more than two times each. Thus the "rule of three" applies.
Theme changes require a restart of GE anyway.